src/main/java/leto/tasklist/Deadline.java

```diff
@@ -0,0 +1,71 @@
+package leto.tasklist;
+
+import java.time.LocalDate;
+import java.time.Period;
+import java.time.temporal.ChronoUnit;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Deadline extends Task {
+    private LocalDate deadline;
+
+    private Deadline(Boolean completed, String TaskString, LocalDate deadline) {
+        // Call to 'super()' must be first statement in constructor body
+        super(completed, TaskString);
+        this.deadline = deadline;
+    }
+
+    public static Deadline DeadlineFromCMD(String input) throws InvalidTaskException {
```

Comment:
Might be better to rename as  `deadlineFromCMD`
I noticed the same issue in several other places too

Reply:
Thanks for catching that!